src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -669,6 +671,24 @@ bool mongoLocationCapable(void)
 
 
 
+/* ***************************************************************************
+*
+* mongoExpirationCapable -
+*/
+bool mongoExpirationCapable(void)
+{
+  int mayor;
+  int minor;
+
+  /* TTL (Time To Live) indexes was introduced in MongoDB 2.2,
+   *  although the expireAfterSeconds: 0 usage is not shown in documentation until 2.4 */
+  mongoVersionGet(&mayor, &minor);
+
+  return ((mayor == 2) && (minor >= 4)) || (mayor > 2);
```

Comment:
`mayor >= 2`, perhaps ?

Reply:
I think the `= 2` case for mayor is already covered in the left part of the `||`

Reply:
Of course. I was too fast :)

NTC